### DISCUSSION TITLE
Next Image with Cloudflare Load Balancer

### DISCUSSION THREAD

**Discussion by mohux** [vercel/next.js#27199] [Help]
Did anyone have an issue where pushing nextjs with next/image with load balancers

the images become slow and sometimes not working?

could it be because of nginx? or because of next/image component it self?

**Comment by brancooo1** [comment]
From my experience (nginx on DigitalOcean), the next/image was slow also. I think there's something wrong with the component itself. Instead I compress it myself, when it's static..

```
<Image
  {...image}
  alt={title}
  priority   // as documentation states, use this if image is above fold, and you wanna prioritise it
  unoptimized  // this disables nextjs optimisations, was the only solution in some cases
/>
```

Hope this helps.

**Reply by mohux** [reply]
What makes no sense that it only breaks when load balancers are implemented

Priority doesnt fix the issue
While unoptimizing isnt tested
But making all images unoptimized, isnt that bad ?

Btw it happens for both local and remote images
